common/converters.go

```diff
@@ -49,20 +49,19 @@ func StringToL2TxState(txState string) (*PoolL2TxState, error) {
 }
 
 // StringToIdx converts string to account index
-func StringToIdx(idxStr, name string) (*Idx, error) {
+func StringToIdx(idxStr, name string) (*Idx, string, error) {
```

Comment:
Maybe there is better to return a struct like this one?
```
type Account struct {
  accountIndex *Idx
  symbol string
}
```

This way the code will be more clear and understandable, and you will not need to use a skip operator.